tests/IntegrationTests/CommandTimeoutTests.cs

```diff
@@ -72,13 +77,16 @@ public void CommandTimeoutWithSleepSync()
 		Assert.Equal(connectionState, m_connection.State);
 	}
 
-	[SkippableFact(ServerFeatures.CancelSleepSuccessfully | ServerFeatures.Timeout)]
-	public async Task CommandTimeoutWithSleepAsync()
+	[SkippableTheory(ServerFeatures.CancelSleepSuccessfully | ServerFeatures.Timeout)]
+	[InlineData(true)]
+	[InlineData(false)]
+	public async Task CommandTimeoutWithSleepAsync(bool prepare)
 	{
 		var connectionState = m_connection.State;
 		using (var cmd = new MySqlCommand("SELECT SLEEP(120);", m_connection))
 		{
 			cmd.CommandTimeout = 2;
+			if (prepare) cmd.Prepare();
```

Comment:
```suggestion
			if (prepare) await cmd.PrepareAsync();
```

Seems reasonable to test the async code path in this test method.